### PR TITLE
Add layout attribute as metadata

### PR DESCRIPTION
**Pull request by paraita**

### CODE REVIEW

**Comment by paraita** [comment]
@ow2-proactive/developers can I have 2 reviews please ?

File: src/main/java/org/ow2/proactive/workflow_catalog/rest/service/WorkflowRevisionService.java [hunk 1]
```diff
@@ -126,11 +126,13 @@ public WorkflowMetadata createWorkflowRevision(Long bucketId, Optional<Long> wor
     @Transactional
     public WorkflowMetadata createWorkflowRevision(Long bucketId, Optional<Long> workflowId,
             ProActiveWorkflowParserResult proActiveWorkflowParserResult,
+            Optional<String> layout,
             byte[] proActiveWorkflowXmlContent) {
         Bucket bucket = findBucket(bucketId);
 
         Workflow workflow = null;
         WorkflowRevision workflowRevision;
+        String layoutValue = layout.isPresent() ? layout.get() : "";
```
Comment (marcocast) [inline src/main/java/org/ow2/proactive/workflow_catalog/rest/service/WorkflowRevisionService.java]: You can use the orElse functionality from the Optional monad here : 

String layoutValue = layout.orElse("")

**Comment by marcocast** [comment]
Seems ok to me

File: src/integration-test/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowControllerIntegrationTest.java [hunk 2]
```diff
@@ -74,6 +74,10 @@
 
     private static final String WORKFLOW_RESOURCE = "/buckets/{bucketId}/workflows/{workflowId}";
 
+    private static final String LAYOUT =
```
Comment (laurentpellegrino) [inline src/integration-test/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowControllerIntegrationTest.java]: I would have renamed all references to layout name to a new name like _layoutMetadata_ to better assess there is nothing related to a UI element associated to the workflow catalog.

**Comment by paraita** [comment]
Thanks guys for your comments :+1:

**Pull request merged**